### PR TITLE
bump API to 1.28

### PR DESCRIPTION
**Pull request by vieux**
1.27 is used in 17.03.1, so bumping master to 1.28

### CODE REVIEW

File: api/server/router/container/container_routes.go [hunk 1]
```diff
@@ -522,7 +522,7 @@ func (s *containerRouter) wsContainersAttach(ctx context.Context, w http.Respons
 		conn := <-wsChan
 		// In case version is higher than 1.27, a binary frame will be sent.
```
Comment (tiborvass) [inline api/server/router/container/container_routes.go]: @vieux please update version in comment too

**Comment by tonistiigi** [comment]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
Failure looks to be genuine (failing on all checks), not sure what's causing it though;

```
04:34:34 ----------------------------------------------------------------------
04:34:34 FAIL: docker_api_images_test.go:16: DockerSuite.TestAPIImagesFilter
04:34:34 
04:34:34 docker_api_images_test.go:40:
04:34:34     c.Assert(images[0].RepoTags, checker.HasLen, 2)
04:34:34 ... obtained []string = []string{"debian:jessie"}
04:34:34 ... n int = 2
04:34:34 
```

**Comment by vdemeester** [comment]
@thaJeztah https://github.com/docker/docker/blob/master/api/server/router/image/image_routes.go#L258 is the reason 😓 
This line should be update too (not sure which API version we should bump it into though..) — it also might be the case for other `versions` condition based on future API version that would break our depreciation policy…

**Comment by vieux** [comment]
@vdemeester I updated, changed the if to avoid the issue

**Pull request merged**